test/view/screens/repositories_screen_test.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:git_hub/core/http/http_exception.dart';
+import 'package:git_hub/models/get_repositories/repository_owner_response.dart';
+import 'package:git_hub/models/get_repositories/repository_response.dart';
+import 'package:git_hub/provider/repositories_provider.dart';
+import 'package:git_hub/repository/get_repositories_repo.dart';
+import 'package:git_hub/view/screens/repositories_screen.dart';
+import 'package:git_hub/view/widget/drop_down.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:provider/provider.dart';
+
+import 'repositories_screen_test.mocks.dart';
+
+@GenerateMocks([GetRepositoriesRepo])
+void main() {
+  group("$RepositoriesScreen", () {
+    late RepositoriesProviderImpl repositoriesProvider;
+    late MockGetRepositoriesRepo mockGetRepositoriesRepo;
+
+    setUp(() {
+      mockGetRepositoriesRepo = MockGetRepositoriesRepo();
```

Comment:
Here you are mocking the wrong layer

Now:
Screen -> Provider -> Repo (mocked)

Expected:
Screen -> Provider (mocked)

Reply:
done